### PR TITLE
fix for /app not working in Page

### PR DESCRIPTION
**Pull request by RaeChen07**

### CODE REVIEW

File: src/components/root_layout.tsx [hunk 1]
```diff
@@ -27,21 +27,26 @@ const RootLayout: FC<RootLayoutProps> = ({ children }) => {
 // back button should send the user to the JobsView. templateName set to the empty string while 
 // showBackButton is true indicates that the back button should send the user to the Home screen.
 const [templateName, setTemplateName] = useState("");
-const [showBackButton, setShowBackButton] = useState(false);
+const [showBackButtonToTemplate, setShowBackButtonToTemplate] = useState(false);
```
Comment (charliepnnl) [inline src/components/root_layout.tsx:30]: I don't think you need to change the state variables.  Instead, just change the <Link to=... code.
```
to={templateName ? `app/{templateName}` : '/'}
```

**Review by charliepnnl — CHANGES_REQUESTED** [review]

File: src/components/root_layout.tsx [hunk 2]
```diff
@@ -26,47 +26,33 @@ const RootLayout: FC<RootLayoutProps> = ({ children }) => {
...
-        setShowBackButtonToHome(false);
         const [, capturedTemplateName] = toTemplateMatchResult;
-        setTemplateName(capturedTemplateName);
-    }  else if (regexPatternToHome.test(location.pathname)){
-        setShowBackButtonToHome(true);
-        setShowBackButtonToTemplate(false);
-        setTemplateName("");
+        setBackUrl("\/app\/"+capturedTemplateName);
+    } else if(regexPatternToHome.test(location.pathname)){
+        setShowBackButtonToTemplate(true);
```
Comment (charliepnnl) [inline src/components/root_layout.tsx:41]: Wouldn't it be clearer to just call this state variable `showBackButton`? The back button doesn't always go to the template.

**Review by charliepnnl — CHANGES_REQUESTED** [review]

File: src/components/root_layout.tsx (hunk 1 above)
Comment (RaeChen07) [reply src/components/root_layout.tsx:29]: I didn't think about this, this is way easier, but it might cause problem if we later have more back button

Comment (charliepnnl) [reply src/components/root_layout.tsx:29]: I really like your use of the `backUrl` state variable. That provides any flexibility we may need in the future. Good idea.

**Review by charliepnnl — APPROVED** [review]
Nice

**Pull request merged**